### ISSUE TITLE
cleanup: Don't ship XSLTs in the tarball that are not needed

### ISSUE THREAD

**Issue by ghost** [openSUSE/daps#612]
I looked in the repo and found that there is a considerable number of XSLT files that do not seem to be used in DAPS at all (the following list.
Can we make a decision on which of these files...
1. Are in some way needed as part of the package? E.g. because they are used by dependent processes that can't be changed easily. (The only file I know to be in this category is `daps-xslt/relnotes/yast.xsl` which is directly referenced by all release notes Makefiles starting with SLES 12 GA.)
2. Which can be deleted entirely? (Some ideas on that in #609, #610, #611)
3. Which can be moved to the repos `contrib/` dir? (not `daps-xslt/contrib`, because that makes them harder to separate from the rest of the source!) 

(@tomschr Thoughts?)

```
daps-xslt/common/catalog_xml2sgml.xsl
daps-xslt/common/extract-components.xsl
daps-xslt/common/get-bookstructure.xsl
daps-xslt/common/get-booktitle.xsl
daps-xslt/common/get-headlines-ids.xsl
daps-xslt/common/get-remarks.xsl
daps-xslt/common/get-rootids.xsl
daps-xslt/common/get-xml-model.xsl
daps-xslt/common/pretty-print.xsl
daps-xslt/common/remove-db5-ns.xsl
daps-xslt/common/resolve-pi.xsl
daps-xslt/common/search4includedfiles.xsl
daps-xslt/contrib/change-productname.xsl
daps-xslt/contrib/checkconformance.xsl
daps-xslt/contrib/convert-table2variablelist.xsl
daps-xslt/contrib/get-graphics-csv.xsl
daps-xslt/contrib/get-ids.xsl
daps-xslt/contrib/get-novell-ids.xsl
daps-xslt/contrib/get-textonly.xsl
daps-xslt/contrib/get-xref2booklist.xsl
daps-xslt/contrib/get-xrefs.xsl
daps-xslt/contrib/listbook-xrefs.xsl
daps-xslt/contrib/maketoc.xsl
daps-xslt/contrib/obfuscate.xsl
daps-xslt/contrib/onlystructure.xsl
daps-xslt/contrib/preserve-remark.trans.xsl
daps-xslt/contrib/simple-headlines.xsl
daps-xslt/contrib/svn.getrevision.xsl
daps-xslt/migrate/db5-move-blocks-outof-para.xsl
daps-xslt/migrate/db5togeekodoc.xsl
daps-xslt/migrate/openstack-db5to4.xsl
daps-xslt/migrate/suse-upgrade.xsl
daps-xslt/pandoc/db2db.xsl
daps-xslt/profiling/indexranges.xsl
daps-xslt/profiling/process-xrefs.xsl
daps-xslt/relnotes/rn-remove-entities.xsl
daps-xslt/relnotes/yast.xsl               [I know this one is used by release notes packages directly]
daps-xslt/xhtml/generate.manifest.xsl
```

**Comment by tomschr** [comment]
Most of them are just helpers to solve a specific problem. We could probably remove them from our RPM package.
However, maybe we should keep at least the stylesheets from the `migrate/` directory.

**Comment by ghost** [comment]
Exactly, they help with specific problems but there is pretty much no reason to ship them as part of DAPS or even put them in the same repo -- in fact, moving them into a separate repo might actually be better, because then people can just clone the small-helpers repo rather than the entire DAPS repo.

**Comment by fsundermeyer** [comment]
Actually, I would prefer to keep them in DAPS and to continue shipping them. Moving them into a seperate repository will add yet _another_ repo aand I have _already_ lost track of a lot of our repositories. I assume I am not the only one... .
Also I sometimes use them for debugging or development.

Shipping them with DAPS allows to use them for debugging or reporting--they are all DocBook-specific after all. I would see this as an added service.

In case we decide to keep these files:  Whether /usr/share/daps is the right place to store these additional files is something worth discussing.

**Comment by ghost** [comment]
When people need to use these kinds of stylesheets, quite often the stylesheets need adjustments for a slightly different use case anyway and then people need to use the version from the DAPS repo already, so they can get updated versions more easily. So, even if they are left in the DAPS repo, I would very much advocate moving them to `[REPOROOT]/contrib` and not shipping them in the RPM.
